forms-flow-api/src/formsflow_api/resources/form_bundling.py

```diff
@@ -0,0 +1,93 @@
+"""API endpoints for managing form bundle resource."""
+
+from http import HTTPStatus
+
+from flask import current_app, request
+from flask_restx import Namespace, Resource
+from formsflow_api_utils.utils import auth, cors_preflight, profiletime
+
+from formsflow_api.schemas import FormBundleProcessMapperSchema
+from formsflow_api.services import FormBundleService, FormProcessMapperService
+
+API = Namespace("bundle", description="Bundle flow")
+
+
+@cors_preflight("GET,POST,OPTIONS")
+@API.route("/create", methods=["GET", "POST", "OPTIONS"])
```

Comment:
POST and GET doesn't need /create. It can be empty then it would appear as POST /bundles